### ISSUE TITLE
declaration-colon-space-after

### ISSUE THREAD

**Issue by timkraut** [stylelint/stylelint#1752]
Feature request for declaration-colon-space-after: Aligned property values are reported as errors. There should be an option to allow 1+ spaces after the colon.

Example:
```css
* {
  padding: 0;
  margin:  0; /* 2 spaces after margin */
}
```

Stylelint configuration:
```json
{
  "rules": {
    "declaration-colon-space-after": "always",
  }
}
```

Stylelint version: `7.0.0` used via gulp-stylelint (`v3.0.0`)

**Comment by evilebottnawi** [comment]
@timkraut duplicate https://github.com/stylelint/stylelint/issues/769?

**Comment by timkraut** [comment]
@evilebottnawi Yes, it's a duplicate. I did a search but unfortunately, I haven't found this issue. Sorry. I'm gonna close this one.
